docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -1 +1,131 @@
+
+== Creating your applications and components with {ProductName}:
+
+At the time of publication, to create applications in {ProductName}, you need to manually maintain their configurations by editing YAML files. This might change, in favor of maintaining those configurations in the {ProductName} UI.
+
+.Prerequisites:
+
+* link:https://kubernetes.io/docs/tasks/tools/[kubectl] CLI tool
+* The following values from your {ProductName} GitHub App:
+** The private key
+** The App ID
+** The webhook secret
+* A GitHub repository with source code for your application 
+** It must have the same basic structure as our example repository
+* Read/write access to the repository for your instance of {ProductName} 
+
+.Procedure:
+
+. In the GitHub repository for your application, install your organization’s {ProductName} GitHub App.
+. In the repository for your instance of {ProductName}, prepare an `application-and-component.yaml` file for your namespace. 
+.. If your namespace does not already have this YAML file, create it with the following path: `/test/resources/demo-users/user/<namespace>/application-and-component.yaml`. Use the namespace 2 file as a template.
+.. Replace the two `url` values in that YAML file as follows:
+... For the `url` value beneath `git`, provide the URL that you would use to locally clone your application’s repository. This URL ends with `.git`.
+... For the `url` value beneath `spec`, at the end of the file, provide the URL for your application’s repository that you see in your browser, without `.git` at the end.
++
+NOTE: To add other components to the same namespace, reference the example YAML file beneath this procedure. 
+. Login to the Kubernetes cluster where Konflux is deployed. 
+. Run the following command. Be sure to replace <namespace> with your actual namespace.
++
+`kubectl create -f /test/resources/demo-users/user/<namespace>/application-and-component.yaml`
+. Run the following command. But first, be sure to export as variables or hard code the values for `PATH_PRIVATE_KEY`, `APP_ID`, and `WEBHOOK_SECRET`.
++
+[source]
+--
+kubectl -n pipelines-as-code create secret generic pipelines-as-code-secret \
+       --from-literal github-private-key="$(cat $PATH_PRIVATE_KEY)" \
+       --from-literal github-application-id="APP_ID" \
+       --from-literal webhook.secret="WEBHOOK_SECRET"
+--
+. Still in the context of your {ProductName} repository, run `kubectl create -f smee/smee-client`.
+. Now, you can trigger your application’s first build pipeline. In the git repository for your application, using your preferred text editor, open a pull request against the `/.tekton/pull-request.yaml` file. 
+.. Specifically, replace any existing value for the `git-url` field with the git URL for your application’s repository. (This is the URL you would use to clone the repository locally; it ends with `.git`.)
```

Comment:
This looks like the process to create the GitHub application, not to create Components and Applications.